src/libServer/Server.cpp

```diff
@@ -360,6 +360,49 @@ Json::Value Server::GetSmartContractState(const string& address)
         return _json;
     }
 }
+
+Json::Value Server::GetSmartContractCode(const string& address)
+{
+    LOG_MARKER();
+
+    try
```

Comment:
Why is whole block in a `try` block?

Reply:
In case of any unforeseen error...similar to other APIs